src/octoprint/server/__init__.py

```diff
@@ -1653,6 +1659,10 @@ def _setup_login_manager(self):
 		if self._settings.get(["accessControl", "trustBasicAuthentication"]):
 			loginManager.request_callback = load_user_from_request
 
+		# Login via REMOTE_USER header
+		if self._settings.get(["accessControl", "useRemoteUser"]):
```

Comment:
This should probably also say `trustRemoteUser`.

Reply:
There's probably a better way to do this than I did but I'm not the best with Python.